### PR TITLE
Add a new ColumnsGuide

### PR DESCRIPTION
**Pull request by thejud**
Per issue https://github.com/saulpw/visidata/issues/2313 . The ColumnsGuide describes how to rename, hide and resize columns.

### CODE REVIEW

File: visidata/guides/ColumnsGuide.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Rename columns to be valid python identifiers that can be used in column
+expressions.
+
+- {help.commands.normalize_col_names}
+- {help.commands.clean_names}
+
+# Hiding and Unhiding columns
+
+Clean up the sheet by hiding unwanted columns.  Hidden columns are still present, just not visible.
```
Comment (saulpw) [inline visidata/guides/ColumnsGuide.md]: In the upcoming version, hidden columns are 1 character columns on the right, which can be unhidden with `_`.

Comment (thejud) [reply visidata/guides/ColumnsGuide.md]: interesting! I see how that would make it much easier to understand what is going on.

**Review by anjakefala — APPROVED** [review]
I really appreciate how thoroughly you covered the features related to column visibility! It looks great in VisiData. 

Screenshot from 2024-09-22 18-25-58

I'm wondering if we want to manually type `resize-col-max` so that we can add an `(underscore)` to the right of the keystroke. Does it read clearly as an underscore?

**Comment by saulpw** [comment]
> I'm wondering if we want to manually type `resize-col-max` so that we can add an `(underscore)` to the right of the keystroke. Does it read clearly as an underscore?

I think it's fine for now.

**Pull request merged**